Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2105,12 +2105,7 @@ private let schemesAllowedToBeOpenedAsPopups = ["http", "https", "javascript", "
 
 extension BrowserViewController: WKUIDelegate {
     func webView(_ webView: WKWebView, createWebViewWith configuration: WKWebViewConfiguration, for navigationAction: WKNavigationAction, windowFeatures: WKWindowFeatures) -> WKWebView? {
-        /* Fix for #446, when in private mode the WKProcessPool of the configuration is changed,
-         this leads to a crash for popups ie window.open where the configuration is created by parent wkwebview.
-         To handle this for now we are stopping popup when in private mode.
-         The fix is done in guard statement below.
-         */
-        guard let parentTab = tabManager[webView], !parentTab.isPrivate else { return nil }
+        guard let parentTab = tabManager[webView] else { return nil }
```

Comment:
Since https://github.com/brave/brave-ios/pull/1007/files#diff-75a5bde6af31e1ad1834bee74125b792L12 is removed, we can now allow popups in private mode so removed unnecessary check.